Dockerfile

```diff
@@ -0,0 +1,39 @@
+# STEP 1 build executable binary
+FROM golang:1.10 as builder
```

Comment:
As with operator images, we don't do the build in here. We do a make target which builds first and stages the jujud so it can be packed into a docker image after it's built.

Reply:
removed